### ISSUE TITLE
Narrator/NVDA not announcing "Copied" status once after click on copy button.:A11y_Awesome AZD_Filter Results_Viewing templates_Screen Reader.

### ISSUE THREAD

**Issue by Harika1702** [Azure/awesome-azd#350]
### Tags
#A11yTCS; #A11ySev3; #A11yMAS; #WCP; #CELA P3; #ChromiumEdge; #FTP; #Win11; #awesome-azd-wcp; #CELA_AwesomeAZD_Web_Mar2024; #MAS1.3.1; #Info and Relationship; #AILimited; #DesktopWeb; #Element:ToolTip; #ScreenReader; #NVDA; #Narrator; #STP; #Chrome;

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Actual Result:
Narrator/NVDA not announcing "Copied" status once after click on copy button.

### Expected Results:
Narrator/NVDA should announce "Copied" status once after click on copy button.

### Environment Details:
Application: Awesome AZD
URL: https://azure.github.io/awesome-azd/
Chromium Edge: Version 123.0.2420.20
Chrome Version 122.0.6261.131 (Official Build) (64-bit)
Narrator/NVDA Version 2023.3.3
Windows 11 Enterprise Version 23H2 OS build 22631.2050.
### Repro Steps:

1. Turn on narrator/NVDA.
2. Hit this URL: https://azure.github.io/awesome-azd/.
3. Tab till “View templates" section.
4. Tab till "Copy" button present in the Authored card and press enter.
5. Now verify the issue that Narrator/NVDA not announcing "Copied" status once after click on copy button or not.
### User Impact:
Screen reader users will not be able to know the information is copied or not once after clicking enter on copy button.
### Recommendations:
Refer [Developer Resources (sharepoint.com).](https://microsoft.sharepoint.com/sites/accessibility/SitePages/Developer-Resources.aspx)
### MAS reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

### Attachment:

**Comment by hemarina** [comment]
@Harika1702 Could you check if these old accessibility issues still exist? We recently made a refresh on UI.

**Comment by hemarina** [comment]
Closing this issue as we made a whole update on the site, and this issue is not update to date with existing site. @Harika1702 Feel free to reopen this issue if needed.
